webapp/src/shared/components/Header/Header.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import { Color } from '../../types';
+import { SmallAvatar } from '../Avatar/Avatar';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import Status, { StatusVariant } from '../Status/Status';
+import './Header.css';
+
+type HeaderProps = {
+  navigationFigure: string;
+  navigationUrl: string;
+  statusVariant?: StatusVariant;
+  children: string;
+};
+
+export default function Header({
+  navigationFigure,
+  navigationUrl,
+  statusVariant,
+  children,
+}: HeaderProps) {
+  return (
+    <header className="header">
+      <div className="header-navigation">
+        <Link to={navigationUrl}>
+          {navigationFigure in IconVariant ? (
+            <Icon
+              variant={navigationFigure as IconVariant}
+              size={IconSize.SMALL}
+              color={Color.LIGHT}
+            />
+          ) : (
+            <SmallAvatar url={navigationFigure} />
+          )}
+        </Link>
+      </div>
+      <div className="header-text">{children}</div>
```

Comment:
I don't think this looks good.
We have a seriffed font for everything else, but this header has its own style. Somehow it is not acquiring the `--primary-font:'DM Sans', sans-serif;` style from the .root, but if you hardcode it as `font-family:'DM Sans;'` you can see the difference. Why don't you use some of our text components? Why did you create them for otherwise?

Reply:
Well spotted, thank you! I think it's fixed now.

Reply:
There was already a task to fix that -> https://github.com/42AGV/ft_transcendence/issues/101. I have checked the points fixed by @gbudau